Docs/Generali/glossario/01_intro.tex

```diff
@@ -1,74 +1,260 @@
 % Insert content here
 \section{A}
-    \subsection{Approvazione}
-    Attività nella quale il documento o parti di files sorgente vengono convalidati dal responsabile di progetto.
     \subsection{Agile}
-    Modo di lavoro che si basa sulla suddivisione del progetto in piccoli passi incrementali, in modo 
-    tale da avere un feedback il più velocemente possibile e sistemare quindi più rapidamente i problemi.
-
+    Approccio alla gestione dei progetti che prevede la suddivisione in fasi e sottolinea 
+    l'importanza della collaborazione e del miglioramento continuo tramite feedback. 
+    Il team segue un ciclo di pianificazione, esecuzione e valutazione.
+    \subsection{API}
+    Abbreviazione per \textit{Application Programming Interface}, indica un insieme di 
+    procedure che consentono la comunicazione tra diversi computer, tra diversi 
+    software o tra diversi componenti di software.
+    \subsection{Approvazione}
+    Attività nella quale un documento viene convalidato dal responsabile di progetto.
+    \subsection{Assignee}
+    Letteralmente "Assegnatario", ovvero membro del team a cui è assegnato lo
+    svolgimento di una determinata attività.
+\pagebreak
 \section{B}
-    \subsection{Branch}
-    Feature di github che permette di sviluppare software, nel nostro caso anche documentazione, in modo da creare una suddivisione del lavoro dove ogni modifica non 
-    influenzi direttamente il ramo principale.
-
+    \subsection{Backlog}
+    Lista prioritaria e ordinata di attività, requisiti o funzionalità che devono 
+    essere completati per sviluppare un prodotto o un progetto.
+    Vedi anche \nameref{Product backlog} e \nameref{Sprint backlog}.
+    \subsection{Branch} 
+    In Git, ramo del progetto che permette di sviluppare software, documentazione inclusa, su una linea separata 
+    senza interferire con il ramo principale o altri rami del repository.
+\pagebreak
 \section{C}
+    \subsection{Capitolato}
+    Documento tecnico a cui si fa riferimento per definire le specifiche tecniche 
+    del prodotto che verrà realizzato. Può contenere indicazioni sulle metodologie e
+    tecnologie da adottare per lo sviluppo del prodotto.
     \subsection{Commit}
-    Sono la parte più importante del sistema di versionamento Git ossia delle istantanee o punti di controllo del progetto in un preciso momento.
-    Servono per creare una cronologia permettendo di tornare indietro ad una versione precedente e tracciare ogni azione eseguita da ogni membro del team.
-
+    In Git, istantanea di uno specifico stato del progetto in un determinato momento.
+    Ogni commit ha un codice univoco che lo identifica e include le modifiche apportate ai file,
+    un riferimento allo stato del repository prima del commit e un messaggio descrittivo fornito dallo sviluppatore.
```

Comment:
Scriverei:
In Git, istantanea di uno specifico stato del progetto in un determinato momento.
Ogni commit ha: un codice univoco che lo identifica, un riferimento allo stato del repository prima del commit e un messaggio descrittivo fornito dallo sviluppatore che descrive brevemente le modifiche apportate ai file.